api/src/main/java/org/pmiops/workbench/api/ErrorTestingController.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+@RestController
+public class ErrorTestingController implements ErrorTestingApiDelegate {
+
+  @Override
+  public ResponseEntity<Void> failFourHundred() {
+    return new ResponseEntity<Void>(HttpStatus.BAD_REQUEST);
```

Comment:
In the spirit of making this part of our documentation/coding patterns, a 400 should always be accompanied by an explanation (what made this a bad request?).